rotations/demonhunter/vengeance.lua

```diff
@@ -28,30 +24,31 @@ local exeOnLoad = function()
 end
 
 local _All = {
-	-- keybind
-  { "Infernal Strike", "keybind(lalt)" },
-  { "Sigil of Flame", "keybind(lcontrol)" },
-
-	{ "/stopcasting\n/stopattack\n/cleartarget\n/stopattack\n/cleartarget", { "player.combat.time >= 300", (function() return F('dpstest') end) }},
+	-- Keybinds
+	{ "Infernal Strike", "keybind(lalt)" },
+	{ "Sigil of Flame", "keybind(lcontrol)" },
 }
 
 local _Cooldowns = {
-  { "Lifeblood" },
-  { "Berserking" },
-  { "Blood Fury" },
+	--{ "Lifeblood" }, -- No point for this. Only Blood and Night Elves can be Demon Hunters
```

Comment:
Can we go ahead and remove these alltogether since, as you point out, it's impossible to have these for demon hunters.